### DISCUSSION TITLE
Trouble adding multiple submitters to Docuseal template using API

### DISCUSSION THREAD

**Discussion by mosesmc52** [docusealco/docuseal#317] [Q&A]
Hi, 

I am trying to add multiple submitters to a Docuseal template using the API, but I don't see any examples how to accomplish this task. How should I proceed?

Moses

**Comment by omohokcoj** [comment]
Hi @mosesmc52 
It's possible to define field `role` name in fields[] json:
![image](https://github.com/user-attachments/assets/484d394a-6cfa-4040-88ba-ce9bdd9a454c)
https://www.docuseal.co/docs/api#create-a-template-from-existing-pdf

Let me know if it answers your question

**Comment by mosesmc52** [comment]
Thanks it works!
